port/win32/omrsysinfo.c

```diff
@@ -1576,3 +1576,38 @@ omrsysinfo_get_open_file_count(struct OMRPortLibrary *portLibrary, uint64_t *cou
 	return OMRPORT_ERROR_SYSINFO_GET_OPEN_FILES_NOT_SUPPORTED;
 }
 
+intptr_t
```

Comment:
ok yeah I would move the windows implementation into the default impl and remove the windows version.  Thanks